app/components/ManagementCardList.tsx

```diff
@@ -7,7 +7,7 @@ export function ManagementCardList({
   cards: MANAGEMENT_CARDS_QUERYResult;
 }) {
   return (
-    <ul className="container mx-auto grid grid-cols-2">
+    <ul className="grid grid-cols-2 auto-rows-fr gap-x-6 gap-y-4 w-full">
```

Comment:
A medium screen query that switches this from grid-cols-2 to just flex might be helpful 
like 
`md:flex` 
not sure what the best form is, feel free to play around with it!